uiowa/uiowa
# Restrict hero block and remove collegiate split

# Updates
Resolves #2585 - Updates all `core.entity_view_display.node.page.default.yml` instances to remove access to the Hero block. Also attempts to bring these files as much in line as possible by restricting most block access for coronavirus.uiowa.edu and admissions.uiowa.edu, which were based on the collegiate split.
Resolves #2517 - Disables the collegiate split for all sites where its active and removes the split.

# How to test

For the following sites:
- default
- admissions.uiowa.edu (collegiate + site split w/ `core.entity_view_display.node.page.default.yml`)
- cogscilang.grad.uiowa.edu (collegiate)
- coronavirus.uiowa.edu (site split w/ `core.entity_view_display.node.page.default.yml`)
- uiowa.edu (collegiate + site split)

Do the following:
- `blt ds --site=<site>`
- `drush @site.local uli admin/people`
- Check a few pages to ensure they look okay and that you can edit them via Layout Builder
- Masquerade as a webmaster
- Visit a page and click the 'Layout' tab
- Check each available layout to make sure that you can't add the 'Hero' block in any of them
- Repeat as an editor

For collegiate sites also note that there are no issues with the update hook and subsequent config import.
